README.md

```diff
@@ -67,7 +67,7 @@ If you wish to run any of the projects then clone this repository and go through
 Any instructions related to running a specific project should be found within the README.md of that project.
 
 > **Note**
-> All projects currently use `forc 0.24.4` and `fuel-core 0.10.1`.
+> All projects currently use `forc 0.25.2` and `fuel-core 0.10.1`.
```

Comment:
It may be worth considering adding the SDK version as I've done with Sway-Libs

Reply:
I'm uncertain whether this is a good idea since the `forc` and `core` versions must match however the `sdk` does not. It may be that a new app comes in which requires new `sdk` features but not a newer release of `forc` or `core`. In this case we need to bump all the apps for the tests rather than have the new app use the newer `sdk`.

Reply:
Furthermore, the most recent commit states that the version is `0.25.1` while the tests actually use `0.25.0`. I think this is a bad idea in general and it should be reverted.

Reply:
Arbitrarily selecting a version and using a sliding window is a bit arbitrary... that being said my general approach is if nothing actually requires the version to be bumped and there have been significant enough changes / a couple major releases have been released then we should bump `forc`. The tests follow is needed. This is a looser approach to specifying a minimal version each time.

Reply:
Ultimately I'd like each project to be completely self contained and have it be managed in the same way so that each project does not affect any other project, the decoupling makes PRs easier to handle, and versions are somewhat frequently updated so that nothing lags behind too much while not spending too much time constantly bumping.

Reply:
Yeah I agree that having each project self contained would be easier to mange PRs.  Also reading your first (@Braqzen ) comment I agree with you.  Perhaps a good rule of thumb for updating the SDK versions could be whenever a breaking change is introduced into the rust SDK.